README.md

```diff
@@ -66,6 +66,16 @@ fn main() -> Result<(), Box<dyn Error>> {
 ```
 
 
+Features
+--------
+
+### `no-cid-as-bytes`
+
+Sometimes it is desired that a CID is not accidentally deserialized into bytes. The `no-cid-as-bytes` feature can be enabled in order to error at runtime in such cases.
```

Comment:
```suggestion
Sometimes it is desired that a CID is not accidentally deserialized into bytes. This can happen because the intermediate serde data model does not retain enough information to be able to differentiate between a bytes container and a CID container when there is a conflicting choice to be made, as in the case of some enum cases. The `no-cid-as-bytes` feature can be enabled in order to error at runtime in such cases.
```

Is this right?

Reply:
Yes, that's completely right.